war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -89,6 +90,18 @@ public String readOrganizationName(final String organizationId) {
    }
 
    /**
+    * Reads the organization id according to its name.
+    *
+    * @param organizationName
+    *       name of the organization
+    * @return id of the given organization
+    */
+   public String readOrganizationId(final String organizationName) {
```

Comment:
The description is outdated. It has been changed in [this commit](https://github.com/Lumeer/engine/commit/c16febe143b360da6967004c0b9461fd8727bf32).